client/packages/cli/index.ts

```diff
@@ -159,11 +211,11 @@ class CircuitCLI {
             // @ts-ignore
             let submissionHeight = await this.sdk.circuit.tx.signAndSendSafe(tx)
                 .then(height => {
-                    console.log("Success: Transfer submitted!")
+                    logger.info("Success: Transfer submitted!")
                     return height
                 })
                 .catch(err => {
-                    console.error("Error: Transfer submission failed! Err:", err);
+                    logger.info("Error: Transfer submission failed! Err:", err);
```

Comment:
logger.info or logger.error?

Reply:
Okay but the prefix `Error:` is not needed. We already communicate that transfer failed.
Err should be in logging kwargs